network-api/networkapi/settings.py

```diff
@@ -529,3 +483,6 @@ def show_toolbar(request):
 # Review apps' slack bot
 GITHUB_TOKEN = env('GITHUB_TOKEN')
 SLACK_WEBHOOK_RA = env('SLACK_WEBHOOK_RA')
+
+# REQUIRED FOR AS LONG AS MEZZANINE HAS NOT BEEN FULLY PURGED
+TESTING = False
```

Comment:
where is this used?

Reply:
no idea: it's required by mezzanine for some purpose that we don't care about, since it'll get removed again when we do the second part of this work that takes out mezzanine entirely. Django/Wagtail don't need this value